scripts/create-connected-app.mjs

```
@@ -0,0 +1,252 @@
+    const deployResult = await conn.metadata.deploy(zipBuffer, {
+      singlePackage: true,
+      checkOnly: false
+    });
+
+    let isDone = false;
+    let errorMessage = '';
+    while (!isDone) {
+      const deployStatus = await conn.metadata.checkDeployStatus(
+        deployResult.id,
+        true
+      );
```

ESLint

[error] Unexpected `await` inside a loop.

(no-await-in-loop)